make/test-containers.mk

```diff
@@ -0,0 +1,11 @@
+
+.PHONY: test-containers/check/format
+test-containers/check/format:
+	yq v test-containers.yaml
```

Comment:
Should we document somewhere how to fix the format of the test-containers.yaml if this job fails?

Reply:
:man_shrugging: It's just checking it's valid yaml tbh, if it fails make it valid yaml

Reply:
I thought it was a linter